software/firmware/calibrate.py

```diff
@@ -1,41 +1,68 @@
-from europi import AnalogueInput, MAX_UINT16
-from machine import Pin, PWM, freq
+from machine import Pin, ADC, PWM
+from time import sleep
+from europi import oled, b1, b2
 
-# Overclock for faster calibration
-freq(250_000_000)
+machine.freq(250000000)
```

Comment:
I don't see machine imported, does this still work? Easy fix, just throw `import machine` at the top to be sure.